src/settings/StaffSlips/StaffSlipManager.test.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import {
+  render,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import { EntryManager } from '@folio/stripes/smart-components';
+
+import StaffSlipDetail from './StaffSlipDetail';
+import StaffSlipForm from './StaffSlipForm';
+import StaffSlipManager from './StaffSlipManager';
+
+jest.mock('./StaffSlipDetail', () => jest.fn(() => null));
+jest.mock('./StaffSlipForm', () => jest.fn(() => null));
```

Comment:
We don't really need it, but it's up to you ofc.

Reply:
discussed